e2e/preference/create.spec.ts

```diff
@@ -0,0 +1,58 @@
+import type { PreferenceCreateData } from '@src/clients/preference/create/types';
```

Comment:
Even in these test files, we will maintain the order of imports according to the rest of the project, first import is the tested module, then the configs and lastly the typings

Reply:
Is there any configuration to be done so that it organizes itself in this specific way? When I save the file, it organizes itself the way it is.